packages/menu/src/use-menu.ts

```diff
@@ -18,6 +18,7 @@ import {
   getValidChildren,
   mergeRefs,
 } from "@chakra-ui/react-utils"
+import { useTheme } from "@chakra-ui/system"
```

Comment:
- useMenu shouldn't have to know about `@chakra-ui/system` or the theme. Please consider passing a `direction` prop to `useMenu`.

- Then you can read the direction from `useTheme` within the `Menu` component and pass it's value to the `useMenu` hook.

Reply:
Yes, you need a changeset for the `menu` package. Basically, every package you've changed needs a changeset.